app/src/main/res/values-land/dimens.xml

```diff
@@ -384,10 +385,14 @@
   <dimen name="profile_rename_activity_profile_rename_input_margin_end">168dp</dimen>
   <dimen name="profile_rename_activity_profile_rename_save_button_margin_end">168dp</dimen>
 
+  <!-- Story Fragment- land dimens -->
```

Comment:
Nit
`<!-- StoryFragment -->`

Also always define new values at the bottom.

Reply:
I resolved it.